ross/bearing_seal_element.py

```diff
@@ -442,37 +442,62 @@ def from_table(cls, n, file):
             sys.exit(file + " not found.")
         except xlrd.biffh.XLRDError:
             df = pd.read_csv(file)
+        header_index = -1
+        for index, row in df.iterrows():
+            for i in range(0, row.size):
+                if isinstance(row[i], str):
```

Comment:
To check the header, I think that you should not only check if you have `str`, but also if that string is equal to something (maybe == 'speed'?).
I am saying this because we could have this xl file with some comments on the cell above the actual table.

Reply:
You're right.

Reply:
In your experience, are these tables of damping and stiffness matrices usually too large? With the current changes, the method searches the entire table for a 'kxx' cell. If these tables are too large, this could get inefficient.

Reply:
Usually, they are not too large. I think we won't have problems with this being inefficient.